site/Dockerfile

```diff
@@ -21,5 +21,8 @@ ADD poll.sh /poll.sh
 ADD poll-0.1.sh /poll-0.1.sh
 ADD poll-0.2.sh /poll-0.2.sh
 ADD site.sh /site.sh
+ADD site-www.tar.gz /var/www/html
+RUN chown -R www-data /var/www/html && \
+    chmod -R 555 /var/www/html
```

Comment:
I think this allows execution of any file in html/, which may not be what we want, and we probably only need it on www-data. I think we want something like chmod -R u+rX,g+rX /var/www/html, where X sets execute on dirs only.